README.md

```diff
@@ -1,56 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My-Portfolio
 
-> Description the project.
+> Setup and mobile version skeleton
 
+![screenshot](./Images/Setup-mobile-skeleton.png)
```

Comment:
[OPTIONAL] 
You should take care of the screenshot. It should not include parts and bits not related to your project.  You can find on the internet how to cut your screenshot in order to show only your project image.

- [ ] Replace your screenshot with a more professional image.

Reply:
Thanks for the suggestion.